Yosemite/YosemiteTests/Stores/StatsStoreV4Tests.swift

```diff
@@ -316,9 +316,33 @@ final class StatsStoreV4Tests: XCTestCase {
         XCTAssertEqual(network.queryParameters?.contains(expectedQuantityParam), true)
     }
 
-    /// Verifies that `StatsActionV4.retrieveStats` makes a network request with the given `force_cache_refresh` parameter.
+    /// Verifies that `StatsActionV4.retrieveStats` makes a network request with the given `force_cache_refresh` parameter if `forceRefresh` is true.
     ///
-    func test_retrieveStats_makes_network_request_with_given_force_cache_rerefresh_parameter() {
+    func test_retrieveStats_makes_network_request_with_given_force_cache_rerefresh_parameter_if_forceRefresh_is_true() {
+        // Given
+        let store = StatsStoreV4(dispatcher: dispatcher, storageManager: storageManager, network: network)
+
+        // When
+        let _: Void = waitFor { promise in
+            let action = StatsActionV4.retrieveStats(siteID: self.sampleSiteID,
+                                                     timeRange: .thisMonth,
+                                                     timeZone: .current,
+                                                     earliestDateToInclude: .init(),
+                                                     latestDateToInclude: .init(),
+                                                     quantity: 1,
+                                                     forceRefresh: true) { _ in
+                promise(())
+            }
+            store.onAction(action)
+        }
+
+        // Then
+        XCTAssertEqual(network.queryParametersDictionary?["force_cache_refresh"] as? Bool, true)
```

Comment:
To make the test more readable, would you consider to declare the assertion at the beginning of the test? Sort of:

```diff
+ let containsExpectedQueryParameter = network.queryParametersDictionary?["force_cache_refresh"] as? Bool
...
- XCTAssertEqual(network.queryParametersDictionary?["force_cache_refresh"] as? Bool, true)
+ XCTAssertEqual(containsExpectedQueryParameter, true)
```

The same for `test_retrieveStats_makes_network_request_without_force_cache_rerefresh_parameter_if_forceRefresh_is_false` down below.

Reply:
Done in a160e3c6cd7673dbe9066592bb5b24bd83b730fd